app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -751,13 +751,6 @@ private val Event.wrapper: EventWrapper<*>?
         Event.HaveNoOpenTabs -> EventWrapper<NoExtraKeys>(
             { Metrics.hasOpenTabs.set(false) }
         )
-        Event.HaveTopSites -> EventWrapper<NoExtraKeys>(
-            { Metrics.hasTopSites.set(true) }
-        )
-        Event.HaveNoTopSites -> EventWrapper<NoExtraKeys>(
-            { Metrics.hasTopSites.set(false) }
-        )
```

Comment:
@rocketsroger Are we sure we no longer need these `Event`? I would check with data to see what they would advise here if possible. If we no longer need them, we should also remove their respective `metrics.yaml` entry otherwise right now we are just not reporting anything in `Metrics.hasTopSites`.

Reply:
No, we still use the actual metrics.  I didn't remove that.  Take a look at https://github.com/mozilla-mobile/fenix/blob/62e2fb1aa6c60b4e42b4759e8cdba1f490854420/app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt#L892

Reply:
Gotcha.